src/renderer/components/deposit/add/Deposit.helper.ts

```diff
@@ -27,11 +27,13 @@ import { PoolsDataMap } from '../../../services/midgard/types'
 export const maxRuneAmountToDeposit = ({
   poolData,
   runeBalance,
-  assetBalance
+  assetBalance,
+  thorchainFees
 }: {
   poolData: PoolData
   runeBalance: BaseAmount
   assetBalance: BaseAmount
+  thorchainFees: DepositFees
```

Comment:
No need to pass all fees. 
`thorchainFees: DepositFees`

Pass fee for THORChain inbound tx only:
`thorchainFees: BaseAmount`